infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -184,5 +184,18 @@
     "id": 32,
     "reason": "Unable to deserialize Filter definition.",
     "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.InvalidCloudProviderException",
+    "id": 33,
+    "reason": "The selected Cloud Provider is not valid.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.InvalidRegionException",
+    "id": 34,
+    "reason": "The selected Region is not valid.",
+    "type": "USER"
```

Comment:
Comparing with `CloudProviderConstraintValidator`, we are not consistent, sometimes we say `Region`, sometimes `Cloud Region`